tests/preferences/preference_tests.py

```diff
@@ -161,16 +161,18 @@ def test_get_user_notification_preferences_with_email_prefix(self):
             username="foo",
             user_preferences={
                 "email:notification": {"enabled": True},
-                "email:notification:test": {"enabled": False},
+                "notifications": {"email:notification:test": {"email": False, "desktop": True}},
```

Comment:
that's not how it should be working, having `email` prefixed notification inside `notifications`

Reply:
currently, this is how it works

Reply:
currently we store it in this way :- `"notifications": {"email:notification:assignments": {"email": False, "desktop": True}},` right @thecalcc ?

Reply:
we store it with the notification id and then `email: boolean` & `desktop: boolean`

Reply:
atm we don't store it inside `notifications` right. once we do we shouldn't have there `email:notification` in the name. I only had there this code with `email:notification` prefix to be backwards compatible, but it shouldn't be used in future and definitely not inside `notifications` dict